tests/onnx/quantization/test_graphs.py

```diff
@@ -13,15 +13,26 @@
 
 # pylint: disable=no-member, redefined-outer-name, no-name-in-module
 import pytest
+from unittest.mock import patch
 
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
+from nncf.common.tensor_statistics.aggregator import StatisticsAggregator
 from tests.onnx.quantization.common import infer_model, min_max_quantize_model
 from tests.onnx.quantization.common import compare_nncf_graph
+from tests.onnx.quantization.common import mock_calculate_activation_quantizer_parameters
+from tests.onnx.quantization.common import mock_get_statistics
+from tests.onnx.quantization.common import mock_collect_statistics
 from tests.onnx.models import ALL_SYNTHETIC_MODELS
 from tests.onnx.models import MultiInputOutputModel
 
-
+@patch('nncf.quantization.algorithms.min_max.algorithm.calculate_activation_quantizer_parameters',
```

Comment:
Use patching and mocking in the same manner as it is done in the majority of the rest of the tests. Search the code base for `mocker.patch` or `mocker.spy` for examples.

Reply:
Done